src/DocumentDB.ChangeFeedProcessor/Exceptions/PartitionException.cs

```diff
@@ -6,13 +6,23 @@ namespace Microsoft.Azure.Documents.ChangeFeedProcessor.Exceptions
 {
     using System;
 
-    internal class PartitionException : Exception
+    /// <summary>
+    /// General exception occurred during partition processing.
+    /// </summary>
+    public class PartitionException : Exception
```

Comment:
All (public) exceptions need to have 4 constructors recommended as part of .net guidelines.